### ISSUE TITLE
Validate cluster-name

### ISSUE THREAD

**Issue by joejulian** [samsung-cnct/k2cli#33]
cluster name has a fixed character set that can be used and cannot be longer than 13 characters. Validate this field.

**Comment by coffeepac** [comment]
might want to take a full review of k2 for where we need to filter other names

**Comment by coffeepac** [comment]
blocked by k2recon

**Comment by coffeepac** [comment]
duplicate of k2#321
